README.md

```diff
@@ -1,279 +1,139 @@
+- [Collections](support/guide/jac_language_guide/collections.md)
+- [Nodes](support/guide/jac_language_guide/nodes.md)
+- [Edges](support/guide/jac_language_guide/edges.md)
+- [Graphs](support/guide/jac_language_guide/graphs.md)
+- [Walkers](support/guide/jac_language_guide/walkers.md)
+    - [Traversing a Graph](support/guide/jac_language_guide/traversing_a_graph.md)
+    - [Referencing a Node in Context](support/guide/jac_language_guide/referencing_node_context.md)
+- [Passing Arguments](support/guide/jac_language_guide/passing_arguments.md)
+- [Specifying Operating Context](support/guide/jac_language_guide/specifying_operating_context.md)
+- [Actions](support/guide/jac_language_guide/actions.md)
+    - [Date Actions](support/guide/jac_language_guide/date_actions.md)
+    - [File Actions](support/guide/jac_language_guide/file_actions.md)
+    - [Net Actions](support/guide/jac_language_guide/net_actions.md)
+    - [Rand Actions](support/guide/jac_language_guide/rand_actions.md)
+    - [Request Actions](support/guide/jac_language_guide/request_actions.md)
+    - [Standard Actions](support/guide/jac_language_guide/standard_actions.md)
+    - [Vector Actions](support/guide/jac_language_guide/vector_actions.md)
+    - [Jaseci Actions](support/guide/jac_language_guide/jaseci_actions.md) 
+
+
+## Codelabs and Guides
+
+- [Build a Custom Jaseci Module](support/codelabs/custom_jaseci_module_codelab.md)
+- [CanoniCAI Example](examples/CanoniCAI/CCAI_codelab.md)
+- [Building a Conversational AI System](support/codelabs/canonicai/chapter1.md)
+    - [Chapter 2](support/codelabs/canonicai/chapter2.md)
+    - [Chapter 3](support/codelabs/canonicai/chapter3.md)
+    - [Chapter 4](support/codelabs/canonicai/chapter4.md)
+    - [Chapter 5](support/codelabs/canonicai/chapter5.md)
+    - [Chapter 6](support/codelabs/canonicai/chapter6.md)
+    - [Chapter 7](support/codelabs/canonicai/chapter7.md)
+- [Install Jaseci using Helm](support/helmcharts/jaseci/README.md)
+- [Create AWS EKS Infrastructure for Jaseci using Terraform](support/infrastructure/terraform/aws/README.md)
+- [Create Azure AKS Infrastructure for Jaseci using Terraform](support/infrastructure/terraform/azure/README.md)
+- [Locust Load Test for JASECI](support/locust/README.md)
+- [Setting up Monitoring for JASECI](support/monitoring/README.md)
+- [NER Examples](examples/ner_examples/README.md)
+
+
+## Jaseci Library Reference
+
+- [Jaseci AI Kit](jaseci_ai_kit/README.md)
+    - [CL Summarization](jaseci_ai_kit/jaseci_ai_kit/modules/cl_summer/README.md)
+    - [Bi-Encoder](jaseci_ai_kit/jaseci_ai_kit/modules/encoders/README.md)
+    - [FLair NER](jaseci_ai_kit/jaseci_ai_kit/modules/ent_ext/README.md)
+    - [Fasttext Classifier](jaseci_ai_kit/jaseci_ai_kit/modules/fast_enc/README.md)
+    - [Object Detection](jaseci_ai_kit/jaseci_ai_kit/modules/object_detection/readme.md)
+    - [PDF Extractor](support/guide/jaseci_ai_kit/modules/non_ai/pdf_ext/README.md)
```

Comment:
This is still pointing at support/guide

Reply:
also, i am still seeing duplicate the jaseci_ai_kit readme duplicated exists in support/guide. We don't need those anymore right?

Reply:
PDF Extractor had no readme under jaseci_ai_kit/jaseci_ai_kit so I referenced a version we grabbed earlier. I decided to leave the support/jaseci_ai_kit to guide the folder restructuring for later. There are other markdown files in there which provide descriptions and listings for groups of AI kit items, such as "encoders", etc. I can back it up and remove it altogether if you insist.